frontend/src/component/personalDashboard/PersonalDashboard.tsx

```diff
@@ -81,21 +83,21 @@ export const StyledCardTitle = styled('div')<{ lines?: number }>(
 );
 
 const ActiveProjectDetails: FC<{
-    project: { flags: number; members: number; health: number };
+    project: PersonalDashboardSchemaProjectsItem;
 }> = ({ project }) => {
     return (
         <Box sx={{ display: 'flex', gap: 2 }}>
             <Box sx={{ display: 'flex', flexDirection: 'column' }}>
                 <Typography variant='subtitle2' color='primary'>
-                    {project.flags}
+                    {project.featureCount}
```

Comment:
minor thing: since this is a new endpoint, should we use `flagCount` or stick with `featureCount`? Not sure we reached a conclusion on that part of the renaming anywhere, but we use `feature` all over the shop, so ... 🤷🏼

Reply:
I decided to keep it consistent with the other endpoint I copied it from

Reply:
We can still change our mind as long as the API is in the unstable land